lib/cc/cli/runner.rb

```diff
@@ -56,6 +59,18 @@ def command
           command_name
         end
       end
+
+      private
+
+      def check_version?
+        first_arg = ARGV.first
+        if %w[--check-version].include? first_arg
```

Comment:
At first I wanted to include a short version as well but failed to come up with a good one. Can change but This seem to be a short-lived code so…